File: model/src/main/java/net/artux/pda/model/user/SimpleUserModel.java
package net.artux.pda.model.user;

import java.time.Instant;
import java.util.UUID;

import lombok.Data;

@Data
public class SimpleUserModel {

    private UUID id;
    private String login;
    private String nickname;
    private String avatar;
    private int pdaId;
    private int xp;
    private int achievements;
    private Gang gang;
    private Instant registration;
}

File: model/src/main/java/net/artux/pda/model/user/SimpleUserModel.kt
package net.artux.pda.model.user

import java.io.Serializable
import java.time.Instant
import java.util.UUID

open class SimpleUserModel : Serializable {
    var id: UUID = UUID.randomUUID()
    var login: String? = null
    var name: String? = null
    var nickname: String? = null
    var avatar: String? = null
    var pdaId = 0
    var xp = 0
    var achievements = 0
    var gang: Gang? = null
    var registration: Instant? = null
}